kuksa-client/docs/examples/async-grpc.md

```diff
@@ -34,6 +36,21 @@ async def main():
 asyncio.run(main())
 ```
 
+Besides this there is a solution where you are not using the client as context-manager
+```python
+import asyncio
+
+from kuksa_client.grpc.aio import VSSClient
+
+async def main():
+    client = VSSClient('127.0.0.1', 55555, ensure_startup_connection=True)
+    await client.connect()
+    current_values = await client.get_current_values(['Vehicle.Speed'])
+    print(current_values['Vehicle.Speed'].value)
```

Comment:
Crashes on freshly created databroker (becasue there is no Datapoint attached, current_values['Vehicle.Speed'] is "none"

For minimal example     
```
print(current_values['Vehicle.Speed'])
```

might be a suitbale fix

Reply:
Yeah. Might be a fix, but is not really what we want maybe an if to catch it could be an idea. But maybe it’s ok for example code. The examples before this PR were not „better“ but I guess we want it „better“ ;)

Reply:
Always want it "better", in "real code" I usually do

```python
if values[whatever_path] is not None: #that means there is a datapoint there
    do_stuff_with(values[whatever_path].value)
´´´

was just thinking in terms of minimal example it might be better or also ok, to have less lines, and just print `values[whatever_path]`

I think both is fine, but the current situation (I know it was like this before), that you get an uncaught exception running the examples in typical situation (i.e. a freshly started databroker) is not good